WRQ-16694: Removed `chai` library dependency

ion-andrusciac-lgp opened the pull request:
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Removed `chai` library dependency

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Updated `ui-tests` to use `expect` from `jest`


### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
Enact dependencies had to be pinned to ~4.8.0 in moonstone and its samples

### Links
[//]: # (Related issues, references)
WRQ-16694

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac [ion.andrusciac@lgepartner.com](mailto:ion.andrusciac@lgepartner.com)

ion-andrusciac-lgp committed:
Modified tests to use `expect` from `jest`

ion-andrusciac-lgp committed:
Removed `try-catch` block to display default `jest` errors

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/signed)](https://cla-assistant.io/enactjs/moonstone?pullRequest=184) <br/>All committers have signed the CLA.

daniel-stoian-lgp committed:
Merge branch 'develop' of https://github.com/enactjs/moonstone into feature/WRQ-16694

hong6316 committed:
Merge branch 'develop' into feature/WRQ-16694

ion-andrusciac-lgp committed:
Updated `ui-test-utils` dependency

ion-andrusciac-lgp committed:
Updated `.travis.yml`

daniel-stoian-lgp committed:
Fixed ui tests for expandable list
Fixed unit tests for dropdown and picker
Skipped ui and unit tests for VirtualList

ion-andrusciac-lgp committed:
Updated `package-lock.json`

ion-andrusciac-lgp committed:
Updated `package-lock.json`

ion-andrusciac-lgp committed:
Updated `enact` version from `^4.8.0` to `~4.8.0`.
Reverted changes from unit tests and ui tests.

ion-andrusciac-lgp committed:
Updated `package-lock.json`

hong6316 commented:
Please fix the travis fail.

daniel-stoian-lgp committed:
removed connection with enact develop branch in travis.yml

daniel-stoian-lgp committed:
updated enact dependencies in moonstone samples

hong6316 reviewed (commented):
👍
